patches/build-config-chrome_build.gni.patch

```diff
@@ -10,9 +10,11 @@ index 4bb4a043cbbcb562aed8926867660547cd7f018c..5d31a9398c08843a212c1472114f31f4
  
  declare_args() {
    # Select the desired branding flavor. False means normal Chromium branding,
-@@ -24,3 +25,5 @@ if (is_chrome_branded) {
- } else {
-   branding_path_component = "chromium"
+@@ -26,6 +27,7 @@ declare_args() {
+   } else {
+     branding_path_component = "chromium"
+   }
++  if (brave_branding_path_component != "") { branding_path_component = brave_branding_path_component }
```

Comment:
this patch is no longer needed, we can now just set `branding_path_component` directly in config.js

Reply:
can be done in a follow up

Reply:
https://github.com/brave/brave-browser/issues/7957